src/traverse.ts

```diff
@@ -14,13 +14,22 @@ type Hooks = {
 };
 
 const _traverse = (obj: object, hooks: Partial<Hooks>, path: JsonPath) => {
+  // @ts-ignore
+  // if (obj?.$ref === '#') {
+  //   return;
+  // }
```

Comment:
This is something like an actual fix, where we just ignore the `$ref: #`.

Reply:
this could potentially break some things because `traverse` is used in other spots, outside of stoplight/json